src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.MultipleStatementsCodeGenerator.cs

```diff
@@ -115,11 +115,10 @@ protected override SyntaxNode GetFirstStatementOrInitializerSelectedAtCallSite()
                 protected override SyntaxNode GetLastStatementOrInitializerSelectedAtCallSite()
                     => CSharpSelectionResult.GetLastStatementUnderContainer();
 
-                protected override Task<SyntaxNode> GetStatementOrInitializerContainingInvocationToExtractedMethodAsync(
-                    SyntaxAnnotation callSiteAnnotation, CancellationToken cancellationToken)
```

Comment:
this parameter always referred to an instance singleton.  So no need to pass it (Esp as the instance member was accessible already to sublcasses).